### DISCUSSION TITLE
Use self-signed cert for SSL on windows-latest runner

### DISCUSSION THREAD

**Discussion by redflitzi** [community/community#112762] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi,

As part of a runner job, I have to establish a SSL connection to a special server with a self-signed certificate.
I have a .pem file for this, stored in a secret.

But how can I add this to the trusted certs in the runner "on the fly"? I can write the .pem from the secret to a file, but dont know how to proceed then.
I could only find some instructions how to add a cert with some GUI-oriented Windows desktop tools, but for the windows-latest runner command line/Powershell only, I didn't find a way (or didnt realize it).
Or is there some Action for this?

Thank you very much!

**Comment by redflitzi** [comment, marked as answer]
Ok, I found the solution myself, it is "certutil":

```    
- name: add the .pem cert to trusted store
  run: Certutil -addstore -enterprise -f "Root" "path\to\my_ca.pem"
```

**Comment by bmrony501** [comment]
- name: add the .pem cert to trusted store
  run: Certutil -addstore -enterprise -f "Root" "path\to\my_ca.pem"
